srcpkgs/pillow-simd/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'pillow-simd'
+pkgname=pillow-simd
```

Comment:
please rename the package `python3-pillow-simd` to fit into the naming scheme of python packages.

Reply:
You also need to rename the directory and change the comment line to the new package name

Reply:
And the commit message

Reply:
It seems to be resolved.